uk.co.compucorp.civicrm.tasksassignments/tasksassignments.php

```diff
@@ -258,6 +258,98 @@ function tasksassignments_civicrm_permission(&$permissions) {
 }
 
 /**
+ * Implements hook_civicrm_navigationMenu().
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_navigationMenu/
+ */
+function tasksassignments_civicrm_navigationMenu(&$params) {
+  _tasksassignments_moveCiviCaseAdminSubMenuItemsUnderTaskAdminSubMenu($params);
+}
+
+/**
+ * Moves some of the items of the "Administer > Civi Case" sub menu under the "Administer > Tasks" sub menu
+ *
+ * @param array $params
+ */
+function _tasksassignments_moveCiviCaseAdminSubMenuItemsUnderTaskAdminSubMenu(&$params) {
+  $administerMenuItems = &_tasksassignments_getAdministerMenuItems($params);
+  $menuItemsToClone = _tasksassignments_filterMenuItemsOfAdministerSubMenu($administerMenuItems, 'CiviCase', ['Case Types', 'Case Statuses']);
+
+  _tasksassignments_cloneMenuItemsInAdministerSubMenu($administerMenuItems, $menuItemsToClone, 'tasksassignments_administer', [
+    'Case Types' => 'Workflows',
+    'Case Statuses' => 'Workflows Status'
+  ]);
+  
+  _tasksassignments_deleteAdministerSubMenu($administerMenuItems, 'CiviCase');
+}
+
+/**
+ * Deletes a sub menu of the given name from the Administer main menu
+ *
+ * @param array $administerMenuItems 
+ * @param string $subMenuName
+ */
+function _tasksassignments_deleteAdministerSubMenu(&$administerMenuItems, $subMenuName) {
+  $subMenuID = CRM_Core_DAO::getFieldValue('CRM_Core_DAO_Navigation', $subMenuName, 'id', 'name');
+
+  unset($administerMenuItems[$subMenuID]);
+}
+
+/**
+ * Clones the given set of menu items into the sub menu of the given name.
+ * It also changes the label of the menu items by using a given mapping
+ *
+ * @param array $administerMenuItems
+ * @param array $menuItems
+ * @param array $subMenuName
+ * @param array $labelsMapping
+ */
+function _tasksassignments_cloneMenuItemsInAdministerSubMenu(&$administerMenuItems, $menuItems, $subMenuName, $labelsMapping) {
```

Comment:
either include in the name of the function that is change labels or create new function for that, while the latter option general better following SRP, in this case I think it is enough to change the name : 

```php
_tasksassignments_cloneMenuItemsInAdministerSubMenuAndChangeLabels
```